docs/university.md

```diff
@@ -657,6 +657,22 @@ xskb1 对应 <http://www.auto.uestc.edu.cn/index/xskb1.htm>
 
 </Route>
 
+## 东华大学
+
+### 教务处通知
+
+<Route author="KiraKiseki" example="/dhu/jiaowu/news/student" path="/dhu/jiaowu/news/:type?" :paramsDesc="['默认为 `student`']">
+
+| 学生专栏 | 教师专栏 |
+| -------- | -------- |
+| student  | teacher  |
+
+</Route>
+
+### 最新信息公开
+
+<Route author="KiraKiseki" example="/dhu/xxgk/news" path="/dhu/xxgk/news">
```

Comment:
Missing closure label `</Route>`

Reply:
Is it still necessary for a route without any argument (like Line 674) to add a `</Route>` label? 
Is it similar to `docs/university.md` in Line 11?

Reply:
You need to close `<Route>` tag. It’s clearly stated [in the documentation](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-tian-jia-jiao-ben-wen-dang). 
In line 11 <Route> tag is closed properly but yours don’t.

Reply:
Apologized for my carelessness.
Corrected by using `/>`